doc/api/inspector.md

```diff
@@ -52,6 +52,20 @@ parameter usage.
 
 Return the URL of the active inspector, or `undefined` if there is none.
 
+```js
+// Node.js process was called with --inspect flag.
+require('inspector').url()
+// 'ws://127.0.0.1:9229/whatever'
```

Comment:
Instead of "whatever" can we use "id", "uuid", or some real example ID.

Reply:
Of course; I really prefer leave a "real id", If I put an "real id", should I specify in a comment that "id" is not a constant? Or it is ok to just leave it as it is... (?) Let me know what do you think

Reply:
The [debugger docs](https://github.com/nodejs/node/blob/master/doc/api/debugger.md) use real example output. Let's do that here too.

Reply:
Done! Fixed at 9c75d5a